### DISCUSSION TITLE
How to achieve redirect from non existent url in documentation

### DISCUSSION THREAD

**Discussion by MatRosiek** [jimporter/mike#191] [Q&A] [answered]
Hi,

Suppose i have `https://my_domain/My_Tool_Name/0.2.0/Documentation/modules/Telnet/` URL which leads to newest (0.2.0 version) and it works fine. How to achieve such behaviour that if one would request `https://my_domain/My_Tool_Name/Documentation/modules/Telnet ` which does not exist (because earlier in URL there is a part which tells version) one would be redirected to newest version? Is it possible to configure and if so how can i do it? I read the documentation but I might miss something.
Currently i have configured mike that so if e.g. one request `https://my_domain/My_Tool_Name` then it would redirect him to my `latest` version of docs.

**Comment by jimporter** [comment, marked as answer]
No, that's not possible with mike. See #72.
